### PR TITLE
Ref/emails

### PR DESCRIPTION
**Pull request by isocolsky**

### CODE REVIEW

**Comment by dabura667** [comment]
I made a pull request to your branch with Japanese.

Please merge with this pull request.

**Comment by isocolsky** [comment]
This is great, thank you for your contribution!

File: lib/emailservice.js [hunk 1]
```diff
@@ -85,19 +109,33 @@ EmailService.prototype.start = function(opts, cb) {
   });
 };
 
+EmailService.prototype._compileTemplate = function(template) {
+  var lines = template.split('\n');
+  return {
+    subject: lines[0],
+    body: _.rest(lines).join('\n'),
```
Comment (matiu) [inline lib/emailservice.js]: `_.rest` FTW :)

**Comment by matiu** [comment]
Very nice job! Code looks great.

One question, IIUC, once we have html versions of the email, we just need to drop them on the repository/server on the appropriate path, right? It is not even necessary to  restart the service, correct?

**Comment by isocolsky** [comment]
Thanks! Yes, that is correct. I avoided caching templates for this very reason.

**Comment by matiu** [comment]
> Maybe we could check if there is support for a particular language only when there is a cache fail? 
> What do you think?

yes, that sounds nice, so we dont need to restart the service. But maybe for a later version... ok?

**Comment by isocolsky** [comment]
ok, that makes sense

**Pull request merged**